README.md

```diff
@@ -52,11 +52,11 @@ Note that the caller must pass itself (`$this`) as a parameter to the callee fun
 <?php
 namespace MyHeritage\Friends\Tests\Friendly;
 
-class AnotherFriendlyClass
```

Comment:
Notice I've changed the name of this class as I think it may mislead - it's not 'another friend', as the relationship is not symetric